### ISSUE TITLE
Don't Kill My Standard Notes - A Possible How-To

### ISSUE THREAD

**Issue by onlinemumie** [standardnotes/forum#1295]
For quite some time, I've had an issue with Standard Notes on Android. Now I've stumbled upon a solution that some might find useful. It turned out that the problem wasn't with Standard Notes at all, but Android 9. The solution is probably obvious for a lot of users, but it wasn't for me, so here it is:

I have 2000+ notes, and for the last year or so, whenever I needed to check something on the run in Standard Notes, the app needed to load and decrypt all my notes - an operation than took at least 30 seconds. This wasn't how the app used to work, but nothing I did seemed to solve the problem. I turned off the setting for battery optimization, and changed the appstart setting to keep running in the background. Still, Standard Notes kept shutting down after a while, which resulted in the 30+ second load and decrypt routine most of times when I flipped out the phone or opened my tablet to check on something.

Then I read an advice on [DontKillMyApp](https://dontkillmyapp.com/) meant for Lenovo devices, but it has worked really well on my Huawei phone as well. It's really simple: Press the square button to see last used apps. Find Standard Notes and swipe down. Now, a padlock icon should appear above Standard Notes. This means the process is locked and kept in the foreground. Since I did this, no more 30+ second load and decrypt routine. I use it on both of my Android devices: a Huawei Honor 9 and a Lenovo Tab P10.

Obvious to some, no doubt, and maybe less so for others.

**Comment by JaspalSuri** [comment]
Hi @onlinemumie, thank you for sharing this! While there are various factors at play, since multiple apps could be kept running in the background, have you noticed any significant impact to your phone's battery life at some point after making this change?

**Comment by onlinemumie** [comment]
No, not really. No significant battery drain from what I can tell.

**Comment by JaspalSuri** [comment]
That's great to hear; thanks for getting back to me on that! 🙂
